docker-compose.yaml

```diff
@@ -27,6 +27,7 @@ services:
       REDIS_HOSTNAME: redis
       REDIS_PORT: 6379
       GRANT_PERCENTAGE: 40
+      GRANT_PERCENTAGE_SOLAR: 25
```

Comment:
This would be the first step to adding app configs. 

Theres also a few other files that need changes if you were to add this as an environment variable for Dev/Test etc (**config-map.yaml** and **values.yaml** being the two big examples) but we can do this after UAT etc :)

Its fine to leave for now in this file tho as it doesn't affect anything